.circleci/config.yml

```diff
@@ -9,20 +9,42 @@ jobs:
     #   only:
     #     - master
         
-    docker:
-      # specify the version you desire here
-      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
-      - image: circleci/python:3.6.1
+    # docker:
+    #   # specify the version you desire here
+    #   # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
+    #   - image: circleci/python:3.6.1
       
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
+    #   # Specify service dependencies here if necessary
+    #   # CircleCI maintains a library of pre-built images
+    #   # documented at https://circleci.com/docs/2.0/circleci-images/
+    #   # - image: circleci/postgres:9.4
 
+    docker:
+      - image: circleci/python:3.6.2-stretch-browsers
+        environment:
+          TEST_DATABASE_URL: postgresql://root@localhost/circle_test
+          
+    # Service container image
+    
+      - image: circleci/postgres:9.6.5-alpine-ram
+    
     working_directory: ~/ttt_python
 
     steps:
       - checkout
+      - run: sudo apt-get update
+      - run: sudo apt-get install postgresql-client-9.6
+      - run: whoami
+      - run: |
+          psql \
+          -d $TEST_DATABASE_URL \
+          -c "CREATE TABLE computer_best_moves (
+            id SERIAL PRIMARY KEY,
+            board_spots VARCHAR(9) NOT NULL,
+            computer_marker VARCHAR(1) NOT NULL,
+            best_move INT);
+            ALTER TABLE computer_best_moves ADD CONSTRAINT game_state UNIQUE (board_spots, computer_marker);"
```

Comment:
Can you move this into a script file? This way the script is more reusable & we can separate changes to the script from the configuration file.

Reply:
implemented in https://github.com/olgaboiar/ttt_python/pull/8/commits/64c76fa4568b28717bc38a465fad8c512d1914c4